scripts/launch_cluster.py

```diff
@@ -0,0 +1,642 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Create a Kubernetes cluster on AWS/Aliyun using python sdk.  You should know that this
+# takes an absolute age, allow 20-30 mins.  That's just the speed stuff
```

Comment:
`allow 20-30 mins` what does this mean.

Reply:
it means the whole creation process would take 20-30 mins